Source/Core/VideoBackends/D3D12/D3DQueuedCommandList.h

```diff
@@ -0,0 +1,618 @@
+// Copyright hdcmeta
+// Dual-Licensed under MIT and GPLv2+
+// Refer to the license.txt/license_mit.txt files included.
+
+#pragma once
+
+#include <d3d12.h>
+
+static const UINT s_queueArraySize = 24 * 1024 * 1024;

```

Comment:
Please don't expose variables like this in global scope. This should be within the ID3D12QueuedCommandList class.